src/commands/publish/mod.rs

```diff
@@ -5,8 +5,6 @@ pub mod upload_form;
 
 pub use package::Package;
 
-use route::Route;
```

Comment:
can we keep this? `Route::new` feels better than `route::Route::new`

Reply:
the constructor is likely to get hosed in the next PR, tbh.

Reply:
restored 'use' statement for route module functions to remove this stutter.